wled00/FX.cpp

```diff
@@ -976,29 +947,27 @@ uint16_t WS2812FX::mode_chase_flash_random(void) {
 /*
  * Alternating pixels running function.
  */
-uint16_t WS2812FX::running(uint32_t color1, uint32_t color2) {
-  uint8_t pxw = 1 + (SEGMENT.intensity >> 5);
-  uint32_t cycleTime = 35 + (255 - SEGMENT.speed);
+uint16_t WS2812FX::running(uint32_t color1, uint32_t color2, bool theatre) {
+  uint8_t width = (theatre ? 3 : 1) + (SEGMENT.intensity >> 4);  // window
```

Comment:
I'm not sure I completely understand.
By removing a base function and replacing it with another I had to make minor adjustments to keep visually similar end effect.
Casual user may never notice.
Even for the professional user with time critical effects this change did not make significant deviation and I did not receive any negative feedback.

Reply:
That's not at all what I'm saying, your adjustments are perfect :)

I was referring to the impact of the intensity slider, and whether a higher intensity setting should increase or decrease a section's width. I've since made up my mind though, more intensity == wider makes more sense here, both for consistency and compatibility.